news/6340.feature

```diff
@@ -0,0 +1,4 @@
+Added a new option ``--save-wheel-names`` to command ``wheel``.
```

Comment:
News entries should be in the imperative tone, so "Add a new option..." (see https://pip.pypa.io/en/stable/development/contributing/#contents-of-a-news-entry).